size-auditor/size-auditor.schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Size auditor Configuration",
+  "description": "Configuration file for the Size Auditor tool",
+  "type": "object",
+  "properties": {
+    "sizeTolerance": {
+      "description": "The tolerance limit above which Size Auditor would fail a pull request",
```

Comment:
Consider specifying the unit of measure in the description and perhaps the key itself.